change of App Configuration section in Readme (#1509)
yaptide/ui: README.md

### App configuration

> Currently, app can be configured by setting the following environment variables in .env file in home directory of the project.

Reviewer: ```suggestion
Currently, app can be configured by setting the following environment variables in .env file in the main project directory (same as `package.json` is located).
```

#### Setting communication with backend

>  `REACT_APP_BACKEND_URL` - URL of the API server. e.g.

Reviewer: UI can be deployed on different machine (with different IP) than a backend. During build phase the UI can be configured to talk to given backend instance via `REACT_APP_BACKEND_URL`

 ```
 REACT_APP_BACKEND_URL = https://localhost:8443
 ```

> The Flask backend is listening on port **6000** on a host called yaptide_flask.

Reviewer: ```suggestion
If the backend is deployed as a set of docker containers, then Flask is listening on port **6000** for HTTP requests (HTTPS is supported only via NGINX proxy) on a host called `yaptide_flask`. 
```

> Additionally, the main proxy server listens on port **5000** for plain HTTP and **8443** for HTTPS. Relevant configuration can be found in this [config file](https://github.com/yaptide/yaptide/blob/master/nginx.conf) of backend 

Reviewer: ```suggestion
Additionally, the main NGINX proxy server listens on port **5000** for plain HTTP and **8443** for HTTPS. Relevant configuration can be found in this [config file](https://github.com/yaptide/yaptide/blob/master/nginx.conf) of backend 
```

#### Other configuration options are:
- `REACT_APP_TARGET` - if set to `demo`, app will not require authentication and will be preloaded with demo results (this version is available at <https://yaptide.github.io/web_dev/>)
- `REACT_APP_ALT_AUTH` - if set to `plg`, app will use plgrid authentication
- `REACT_APP_DEPLOYMENT` - if set to `dev`, configuration wil be editable from the browser console. For example, you can change the backend URL by typing `window.BACKEND_URL="http://mynew.url"` in the browser console.

> 
> 

Reviewer: ```suggestion
```

### App configuration

Currently, app can be configured by setting the following environment variables in .env file in the main project directory (same as `package.json` is located).

#### Setting communication with backend

> UI can be deployed on different machine (with different IP) than a backend. During build phase the UI can be configured to talk to given backend instance via REACT_APP_BACKEND_URL
> 
>  `REACT_APP_BACKEND_URL` - URL of the API server. e.g.
>  ```
>  REACT_APP_BACKEND_URL = https://localhost:8443
>  ```

Reviewer: ```suggestion
UI can be deployed on different machine (with different IP) than a backend. During build phase the UI can be configured to talk to given backend instance via `REACT_APP_BACKEND_URL` environmental variable. To adjust it put following line in the `.env` file:

 ```
 REACT_APP_BACKEND_URL = https://localhost:8443
 ```
```
